sopel/test_tools.py

```diff
@@ -327,11 +327,10 @@ class IRCFactory(object):
         This factory is best used with all the testing tools available:
 
         * :func:`botfactory` to create a test instance of Sopel
-        * :class:`UserFactory` to fake IRC user who will join channels, send
+        * :class:`UserFactory` to fake IRC users who will join channels, send
           messages, use commands, trigger URL callback, etc.
-        * :class:`MockIRCBackend` which doesn't need a connection to any real
-          server and will store every messages sent by the bot in a convenient
-          place
+        * :class:`MockIRCBackend` doesn't need a server connection and stores
```

Comment:
I'll remember not to use "[…]" when suggesting stuff like this. I didn't mean to remove the "which" here. 😹

Reply:
lol